bin/liquidator/src/trigger_tcs.rs

```diff
@@ -1225,6 +1234,15 @@ impl Context {
             .instructions
             .append(&mut trigger_ixs.instructions);
 
+        let liqor = &self.mango_client.mango_account().await?;
+        tx_builder.instructions.append(
+            &mut self
+                .mango_client
+                .health_check_instruction(liqor, self.config.min_health_ratio, MaintRatio)
```

Comment:
I did it this way because you can actually check any account (permissionless)
But not sure why we would want to do that, so I'm okay to change that if you want ?

Reply:
both are fine, since all the uses right now pull the mango account before calling the function, I'd probably make a change - but you're right the current version is more flexible, so it's also ok to keep